flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/codegen/EqualiserCodeGenerator.scala

```diff
@@ -29,6 +30,10 @@ import scala.collection.JavaConversions._
 
 class EqualiserCodeGenerator(fieldTypes: Seq[LogicalType]) {
 
+  def this(fieldTypes: Array[LogicalType]) {
```

Comment:
Can we improve the existing constructor to accept only Array[] ? So that we can avoid to maintain two constructors.